src/ecies/ecies_salsa_sealed_box.rs

```diff
@@ -104,14 +104,14 @@ impl Ecies<X25519PrivateKey, X25519PublicKey> for EciesSalsaSealBox {
 #[cfg(test)]
 mod tests {
 
-    use rand_chacha::rand_core::SeedableRng;
-
     use crate::{
-        asymmetric_crypto::{X25519PrivateKey, X25519PublicKey},
+        asymmetric_crypto::{Ed25519PrivateKey, X25519PrivateKey, X25519PublicKey},
         ecies::ecies_salsa_sealed_box::EciesSalsaSealBox,
-        CsRng, Ecies, FixedSizeCBytes, RandomFixedSizeCBytes,
+        CsRng, Ecies, Ed25519PublicKey, FixedSizeCBytes, RandomFixedSizeCBytes,
     };
 
+    use rand_chacha::rand_core::SeedableRng;
```

Comment:
Use the reexport instead?
```suggestion
    use crate::reexport::rand_core::SeedableRng;
```

Reply:
Yes. Fixed the doctest too